docs/user/types/version-usage.en.md

```diff
@@ -1,3 +1,3 @@
 # Version usage
 
-* The version must follow either integer versioning or semantic versioning homogeneously within the same document.
+The version must follow either integer versioning or semantic versioning homogeneously within the same document.
```

Comment:
In the author guide it will at least be linked. For the help panel we still need to figure out some technical details but the plan is to show it, yes.